### DISCUSSION TITLE
Input error messages not read by screen reader

### DISCUSSION THREAD

**Discussion by lonebadger99** [livewire/flux#441] [Feature Requests]
I would like to ask if I am doing something wrong regarding forms and input fields.

Here is an example form:
```
<form wire:submit='submit'>
    @csrf
<flux:card class="max-w-4xl mx-auto px-4 sm:px-6 lg:px-8">
    <flux:fieldset>
        <flux:legend>{{ __('user data') }}</flux:legend>

        <div class="grid grid-cols-1 lg:grid-cols-2 gap-4">
            <flux:input label="name" wire:model="name" description="name" badge="{{ __('mandatory') }}" required autofocus />
            <flux:input label="phone" wire:model="phone" description="phone" badge="{{ __('mandatory') }}" required />
        </div>
    </flux:fieldset>
</flux:card>
    <flux:card class="max-w-4xl mx-auto px-4 sm:px-6 lg:px-8 mt-4">
        <flux:fieldset>
            <div class="grid grid-cols-2 gap-4">
                <flux:button wire:click="showStep('option-step')">{{ __('Back') }}</flux:button>
                <flux:button type="submit">{{ __('Submit') }}</flux:button>
            </div>
        </flux:fieldset>
    </flux:card>
</form>
```

When I submit the form and some server side validation generates an error, I can see the error message in the screen (in the correct place, next to the input field that generated the error) but the screen reader does not announce any errors.

If a blind person is using my app they won´t be able to understand that they need to go to the proper input and change whatever is generating the error. I tried with NVDA and Narrator screen readers.

Am I doing something wrong or is this a bug? Thanks!

**Comment by s-tinga** [comment]
I had the same issue with errors not being announced within a Livewire form while using Flux. I ended up publishing the `flux/error.blade.php` file and then wrapping the div in an `aria-live` region which has worked well for me. It'd be nice if that was a core feature! 

```html
<div aria-live="polite" aria-atomic="true" >
    <?php if ($message) : ?>
        ... default flux error code here ...
    <?php endif; ?>
</div>
```

**Comment by joshhanley** [comment]
@lonebadger99 and @stuininga-pl I'd consider this a bug/ issue not a feature request as Flux is designed to be accessible out of the box. Could I get one of you to raise an issue so this can be investigated? Thanks!
